app/src/main/java/dev/ricsarabia/cryptochallenge/data/services/TickerResponse.kt

```diff
@@ -0,0 +1,18 @@
+package dev.ricsarabia.cryptochallenge.data.services
+
+data class TickerResponse(
+  val success: Boolean,
+  val payload: Payload
+) {
+  data class Payload(
+    val book: String,
+    val volume: String,
+    val high: String,
+    val last: String,
+    val low: String,
+    val vwap: String,
+    val ask: String,
+    val bid: String,
+    val created_at: String
```

Comment:
Here you can use @SerializedName("created_at") and your variable looks like this val createdAt: String :D 
```suggestion
   @SerializedName("created_at")
    val createdAt: String
```